FhirBlaze.PractitionerModule/Components/PractitionerDetailComponent.razor.cs

```diff
@@ -85,7 +85,7 @@ public void AddHumanName()
             {
                 this.Practitioner.Name = new List<HumanName>();
             }
-            this.Practitioner.Name.Add(new HumanName() { });
+            this.Practitioner.Name.Add( new HumanName() { Period = new Period() });
```

Comment:
Is there anything else that should be defaulted?

Reply:
Not currently for practitioner but we should review all the components to be sure.